src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.NotValidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final String VALID_LOGIN = "CheckLogin";
+    private static final String NULL_STRING = null;
+    private static final String INCORRECT_LOGIN = "abcd";
+    private static final String INCORRECT_PASSWORD = "logi";
+    private static final Integer INCORRECT_AGE = 13;
+    private static final String VALID_PASSWORD = "CheckPassword";
+    private static final Integer NULL_AGE = null;
+    private static final int VALID_AGE = 18;
```

Comment:
Please, read Nazar's comment about constants in test class.

Also, don't need to create constant for null parameters